src/mention/mention-list.component.ts

```diff
@@ -52,6 +52,7 @@ export class MentionListComponent implements OnInit {
   items = [];
   activeIndex: number = 0;
   hidden: boolean = false;
+  private blockCursorSize;
```

Comment:
I think this can be removed because it's never set?

Reply:
Oh snap, I was supposed to set it on line 68 where currently I'm setting const. Thanks! Idea is to cache `getBlockCursorDimensions()` result to save calls to `window.getComputedStyle()`. I'll update that.